src/style.css

```diff
@@ -0,0 +1,109 @@
+body {
+  color: #000;
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.product-card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  padding: 30px 15px 15px;
+  width: 200px;
+}
+
+.image-wrap {
+  height: 134px;
+  margin: 0 auto;
+  padding-bottom: 44px;
+  width: 158px;
+}
+
+.image-wrap img {
+  width: 100%;
+  height: auto;
+}
+
+.product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin: 0;
+}
+
+.product-code {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.stars-list {
+  list-style: none;
+  padding-left: 0;
+  display: flex;
+  margin: 0;
+  width: 91px;
+  justify-content: space-between;
+}
+
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 13px 0 19px;
+}
+
+.stars-item {
+  height: 15px;
+  width: 15px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.rating-text {
+  font-size: 11px;
+  line-height: 14px;
+  padding-top: 2px;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  padding-bottom: 20px;
+}
+
+.price-title {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.cost {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+  width: 79px;
+}
+
+.buy {
+  border: 1px solid #00acdc;
+  box-sizing: border-box;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #fff;
+  background: #00acdc;
+  border-radius: 5px;
+  padding: 12px 0;
+  display: block;
+  transition: 0.5s ease;
```

Comment:
Great job. Just a suggestion: define explicitly which property would you like to be animated. 
```css
transition: color 0.5s ease, background-color 0.5s ease;
```
or: 
```css
transition-property: color, background-color;
transition-duration: 0.5s;
transition-timing-function: ease;
```

https://developer.mozilla.org/en-US/docs/Web/CSS/transition